### PR TITLE
feat song first love added

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 03/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    Forgive me first love
+  </h4>
+</div>
+<footer>
+  <h6>
+    <p>Fuente: Musixmatch</p>
+    <p>Compositores: Adele Laurie Blue Adkins</p>
+    <p>Letra de First Love © Melted Stone Publishing Ltd.</p>
+  </h6>
+</footer>
```
Comment (eamzea) [inline 03/index.html:65]: Indentation

File: 03/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    Forgive me first love
+    Forgive me first love
+  </h4>
+</div>
+<footer>
+  <h6>
+    <p>Fuente: Musixmatch</p>
+    <p>Compositores: Adele Laurie Blue Adkins</p>
+    <p>Letra de First Love © Melted Stone Publishing Ltd.</p>
+  </h6>
```
Comment (eamzea) [inline 03/index.html:64]: Paragraphs inside heading tags?

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by andysotocruz — COMMENTED** [review]
where is the nav?

**Review by andysotocruz — COMMENTED** [review]
are there only p and h tags for texts?

File: 03/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    Forgive me
+    Forgive me first love
+    Forgive me first love
+  </p>
+</div>
+<footer>
+    <p>Fuente: Musixmatch</p>
+    <p>Compositores: Adele Laurie Blue Adkins</p>
+    <p>Letra de First Love © Melted Stone Publishing Ltd.</p>
+</footer>
```
Comment (eamzea) [inline 03/index.html:65]: Indentation

**Review by andysotocruz — COMMENTED** [review]
again, are there only paragraph and heading tags for text?

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 03/index.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    Forgive me first love
+    Forgive me
+    Forgive me first love
+    Forgive me first love
+  </p>
+</div>
+<footer>
+  <p><i>Fuente: Musixmatch</i></p>
+  <p>Compositores: <strong>Laurie Blue Adkins</strong></p>
+  <p><mark>Letra de First Love © Melted Stone Publishing Ltd.</mark></p>
```
Comment (eamzea) [inline 03/index.html:64]: This can be better organized

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — DISMISSED** [review]

File: 03/index.html [hunk 5]
```diff
@@ -11,137 +11,195 @@
...
+    </p>
+    <p>Compositores: 
+      <strong>Laurie Blue Adkins</strong>
+    </p>
+    <p>
+      <mark>Letra de First Love © Melted Stone Publishing Ltd.</mark>
+    </p>
+  </footer>
+</body>
+</html>
```
Comment (eamzea) [inline 03/index.html:73]: Why this closing tag is here?

File: 03/index.html [hunk 6]
```diff
@@ -11,137 +11,195 @@
...
+      Compositores: Billie Eilish O'Connell / Finneas Baird O'Connell / Khalid Robinson
+      Letra de lovely  
+    </h6>
+    <h6>
       © Kobalt Music Publishing Ltd., Sony/ATV Music Publishing LLC, Universal Music Publishing Group 
-      </h6>
-    </footer>
+    </h6>
+  </footer>
   </main>
```
Comment (eamzea) [inline 03/index.html:204]: I see another closing main tag in line 43

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 03/index.html [hunk 7]
```diff
@@ -11,137 +11,192 @@
...
+      <i>Fuente: Musixmatch</i>
+    </p>
+    <p>Compositores: 
+      <strong>Laurie Blue Adkins</strong>
+    </p>
+    <p>
+      <mark>Letra de First Love © Melted Stone Publishing Ltd.</mark>
+    </p>
+  </footer>
+  <header>Vete de una vez</header>
```
Comment (eamzea) [inline 03/index.html:72]: Plain text inside header tag?

File: 03/index.html [hunk 8]
```diff
@@ -11,137 +11,192 @@
...
+    </p>
+    <p>Compositores: 
+      <strong>Laurie Blue Adkins</strong>
+    </p>
+    <p>
+      <mark>Letra de First Love © Melted Stone Publishing Ltd.</mark>
+    </p>
+  </footer>
+  <header>Vete de una vez</header> 
+    <h2>Daniela Spalla</h2>
```
Comment (eamzea) [inline 03/index.html:73]: Indentation

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 03/index.html [hunk 9]
```diff
@@ -4,144 +4,201 @@
...
+      <i>Fuente: Musixmatch</i>
+    </p>
+    <p>Compositores: 
+      <strong>Laurie Blue Adkins</strong>
+    </p>
+    <p>
+      <mark>Letra de First Love © Melted Stone Publishing Ltd.</mark>
+    </p>
+  </footer>
+  <header>Vete de una vez</header>
```
Comment (eamzea) [inline 03/index.html:72]: The same as the last requested change
Is good to have plain text inside a container tag?

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**